src/Websocket.Client/Websocket.Client.csproj

```diff
@@ -29,16 +29,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LibLog" Version="5.0.8">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
-    </PackageReference>
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.2" />
```

Comment:
you have switched all other (test) projects to netcoreapp3.1, but not this one, why?

Reply:
its lib so it does not need to be

Reply:
I mean that the targets of Websocket.Client.Tests.csproj, Websocket.Client.Sample.csproj, etc. were changed to netcore3.1 but Websocket.Client.csproj still remain on netstand2.0.
What was the reason to move test projects to netcore3.1 then?
I would keep them on the same target (netstand2.0).

Reply:
> What was the reason to move test projects to netcore3.1 then?

net core 2.0 is no longer supported 
https://dotnet.microsoft.com/platform/support/policy/dotnet-core